lib/policyEvaluator/RequestContext.ts

```diff
@@ -15,9 +15,19 @@ import {
     actionMapScuba,
 } from './utils/actionMaps';
 
-const _actionNeedQuotaCheck = {
+export const actionNeedQuotaCheck = {
     objectPut: true,
+    objectPutVersion: true,
     objectPutPart: true,
+    objectRestore: true,
+};
+
+// This variable describe APIs that change the bytes
+// stored, requiring quota updates
```

Comment:
It can be nice to use jsdoc `/** */` to have description in tooltip on hover in vscode